frontend/app/views/fragments/bundle/bundleOfferingsB.scala.html

```diff
@@ -1,3 +1,196 @@
 @import abtests.BundleVariant
+@import views.support.Asset
 
 @(bundleVariant: BundleVariant)
+<section class="bundle-section">
+    <div class="bundle-offering-b l-constrained">
+        <div class="bundle-offering-b__subscribe">
+            <div class="bundle-offering-b__line"></div>
```

Comment:
Can we make this a `<hr>` to be more semantic? (I feel we've discussed this before, but I can't remember the outcome!)